# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/qiv/hotfix-00-config.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- qiv-3.0.2/Makefile.vanilla	2025-08-22 00:51:52.719487378 +0200
+++ qiv-3.0.2/Makefile	2025-08-22 00:53:18.735902078 +0200
@@ -4,7 +4,7 @@
 #######################################################################
 
 # Directory where qiv will be installed under.
-PREFIX = /usr/local
+PREFIX = /usr
 
 # Fonts to use for statusbar and comments
 STATUSBAR_FONT = "Monospace 9"
@@ -16,10 +16,10 @@
 CURSOR = 84
 
 # Should image be centered on screen by default? 1=yes, 0=no.
-CENTER = 1
+CENTER = 0
 
 # Should images be filtered by extension? 1=yes, 0=no.
-FILTER = 1
+FILTER = 0
 
 # This sets the file extentions to filter on (other file types will be
 # skipped.) It should reflect whatever can be loaded via gdk-pixbuf-loader.
@@ -30,7 +30,7 @@
 
 # This program will be run on the manual page after it is installed.
 # If you don't want to compress the manpage, change it to 'true'.
-COMPRESS_PROG = gzip -9f
+COMPRESS_PROG = true
 
 # Comment this line out if your system doesn't have lcms2 installed
 # (for minimal Color Management support)
@@ -169,10 +169,6 @@
 	  $(INSTALL) -d -m 0755 $(PREFIX)/share/applications; \
 	fi
 	$(INSTALL) -m 0644 qiv.desktop $(PREFIX)/share/applications/qiv.desktop
-	@if ./qiv -f ./intro.jpg ; \
-	then echo "-- Test Passed --" ; \
-	else echo "-- Test Failed --" ; \
-	fi
 	@echo "\nDont forget to look into the \"qiv-command\" file and install it!\n-> cp qiv-command.example $(PREFIX)/bin/qiv-command\n\n"
 
 # the end... ;-)
